src/printer.js

```diff
@@ -628,7 +628,23 @@ function genericPrintNoParens(path, options, print) {
 
       return concat(parts);
     case "CallExpression": {
-      const parent = path.getParentNode();
+      if (
+        // We want to keep require calls as a unit
+        n.callee.type === "Identifier" && n.callee.name === "require" ||
+        // `it.only('long name', () => {` should not break
+        (n.callee.type === "Identifier" ||
+          (n.callee.type === "MemberExpression" && n.callee.object.type === "Identifier" && n.callee.property.type === "Identifier")) &&
+          n.arguments.length === 2 &&
+          (n.arguments[0].type === "StringLiteral" || n.arguments[0].type === "Literal" && typeof n.arguments[0].value === "string") &&
+          (n.arguments[1].type === "FunctionExpression" || n.arguments[1].type === "ArrowFunctionExpression") &&
+          n.arguments[1].params.length <= 1
```

Comment:
I'm a little hesitant not restrict it to a whitelist of names. Although it would be super rare, this would make it surprisingly ignore a few other places in the codebase that happen to call a function with the same signature.

What do you think about just whitelisting `it` and `xit` just like we do `require`? I'm pretty sure that would solve 99% of the problems. All of the other names like `describe` and others are used minimally and wouldn't break as often because those are at the top of the files, and don't have long descriptive names. I think just detecting `it` would be ok.